src/modules/commander/Commander.cpp

```diff
@@ -2238,6 +2238,16 @@ Commander::run()
 
 			/* publish vehicle_status_flags */
 			status_flags.timestamp = hrt_absolute_time();
+
+			// Evaluate current prearm status
+			if (!armed.armed) {
+				bool preflight_check_res = PreFlightCheck::preflightCheck(nullptr, status, status_flags, true, true, true, 30_s);
```

Comment:
This should be:
```suggestion
				bool preflight_check_res = PreFlightCheck::preflightCheck(&mavlink_log_pub, status, status_flags, _arm_requirements.global_position, false, true,
				       hrt_elapsed_time(&_boot_timestamp))
```

Reply:
I think the idea was that if you don't have the `mavlink_log_pub` you will not publish repeating messages as an in-between solution, right?

Reply:
Yes